APIs/openEO/Python_Client/Python.qmd

```diff
@@ -0,0 +1,344 @@
+- list data collections, available processes, file formats and other capabilities of the back-end
+- start building your openEO algorithm from the desired data on the back-end
+- execute and monitor (batch) jobs on the back-end
+- etc.
+
+
+### Collections
+
+The EO data available at a back-end is organised in so-called collections.
+For example, a back-end might provide fundamental satellite collections like "Sentinel 1" or "Sentinel 2",
+or preprocessed collections like "NDVI".
+Collections are used as input data for your openEO jobs.
+
+::: {.callout-note}
+
+More information on how openEO "collections" relate to terminology used in other systems can be found in
+([the openEO glossary](../Glossary.qmd#eo-data-collections){target="_blank"}).
+
+:::
+
+If necessary, a more detailed metadata listing for a given collection can be obtained with
+[`describe_collection`](https://open-eo.github.io/openeo-python-client/api.html#openeo.rest.connection.Connection.describe_collection){target="_blank"}.
+
+
+### Processes
+
+Processes in openEO are operations that can be applied on (EO) data
+(e.g. calculate the mean of an array, or mask out observations outside a given polygon).
+The output of one process can be used as the input of another process,
+and by doing so, multiple processes can be connected that way in a larger "process graph":
+a new (user-defined) processes that implements a certain algorithm.
+
+::: {.callout-note}
+
+Check [the openEO glossary](../Glossary.qmd)
+for more details on pre-defined, user-defined processes and process graphs.
+
+:::
+
+For other graphical overviews of the openEO processes, there is an [online documentation](https://docs.openeo.cloud/processes/){target="_blank"} for general process descriptions and the [openEO Hub](https://hub.openeo.org/) for back-end specific process descriptions. 
+
+## Authentication 
+
+In the code snippets above we did not need to log in
+since we just queried publicly available back-end information.
+However, to run non-trivial processing queries one has to authenticate
+so that permissions, resource usage, etc. can be managed properly.
+
+
+A detailed description of why and how to use the authentication methods is on the [official documentation](https://open-eo.github.io/openeo-python-client/auth.html#authentication-and-account-management){target="_blank"}.
+
+
+To authenticate your account on the backend of the Copernicus Data Space Ecosystem, it is necessary for you to complete the registration process. Once registered,  the OIDC ([OpenID Connect](https://openid.net/connect/){target="_blank"})authentication method will be employed to verify your identity using an external service.
+
+The following code snippet shows how to log in via OIDC authentication:
+
+```python
+print("Authenticate with OIDC authentication")
+connection.authenticate_oidc()
+```
+
+Calling this method opens your system web browser, with which you can authenticate yourself on the back-end authentication system. 
```

Comment:
This is bit misleading: calling this method does not automatically open a web browser. When there are not valid refresh tokens, it will print a URL to visit. When there are refresh tokens: these will be used directly